includes/embeds/class-amp-gallery-embed.php

```diff
@@ -16,15 +16,13 @@ class AMP_Gallery_Embed_Handler extends AMP_Base_Embed_Handler {
 	 * Register embed.
 	 */
 	public function register_embed() {
-		add_shortcode( 'gallery', array( $this, 'shortcode' ) );
+		add_filter( 'post_gallery', array( $this, 'override_gallery' ), 10, 2 );
 	}
 
 	/**
 	 * Unregister embed.
 	 */
-	public function unregister_embed() {
-		remove_shortcode( 'gallery' );
-	}
+	public function unregister_embed() {}
```

Comment:
This extends an abstract class, so this method has to be implemented. This isn't ideal, but editing this embed handler enables deleting the entire 'Video' widget subclass.